core/src/main/scala/cats/data/EitherT.scala

```diff
@@ -647,6 +647,11 @@ final case class EitherT[F[_], A, B](value: F[Either[A, B]]) {
         case Left(a)  => Validated.invalidNec(a)
       }
     )
+
+  /** Convert this `EitherT[F, A, B]` into an `IorT[F, A, B]`.
+   */
+  def toIorT(implicit F: Functor[F]): IorT[F, A, B] =
```

Comment:
I am not sure about the name. For example, conversion to `OptionT` has the name  just `toOption` (no `T` at the end). Should we follow the same convention here and name it as just `toIor` instead?

Reply:
@satorg hmm. I had noticed something similar on `IorT` with `toEither: EitherT[F, A, B]`. I had initially assumed this was a typo, but maybe it is an intentional convention. It seems strange to me, as there are two distinct ways view an `EitherT[F, A, B]` as an optional value, `OptionT[F, B]` and `F[Option[B]]` and I could see having both `toOptionT` and `toOptionF`. It seems we are a bit inconsistent right now.

For example, in `IorT.scala` we have [`fromEither`, which operates on `Either[A, B]` as opposed to `EitherT[F, A, B]`](https://github.com/typelevel/cats/blob/main/core/src/main/scala/cats/data/IorT.scala#L312) and [`fromEitherF` which operates on `F[Either[A, B]]`](https://github.com/typelevel/cats/blob/main/core/src/main/scala/cats/data/IorT.scala#L324).

It seems like if we want to standardize on a convention which can work with `A`, `F[A]`, and `G[F, A]` (where `G` is some transformer type), then we have to go with the `F` and `T` suffixes.

Thoughts?

Reply:
@isomarcte I'm not that certain about the reasoning behind that convention, but I _guess_ that it just does not make a lot of sense to have both `toOptionT` returning `OptionT` and `toOptionF` returning `F[Option[...]]`, because `OptionT` implies `F[Option[...]]` already via its `value` field. And seems the same is applicable for `EitherT` and `IorT`.

Reply:
@satorg I'll update it. It does seem to be the convention.

Reply:
@satorg updated.